core/tools/nginx/nginx.conf

```diff
@@ -113,6 +113,10 @@ http {
             proxy_pass http://127.0.0.1:81;
         }
 
+        location /logviewer {
+            root /home/pi/tools;
```

Comment:
Should it be /home/pi/tools/logviewer ?

Reply:
no, the current way the full path ends up at /home/pi/tools/logviewer (root + location)

Reply:
Thanks! That was what I was imagining